From 62f3d0572c54b8d8989c042e6083611d73166178 Mon Sep 17 00:00:00 2001
From: Rex Zhu <Rex.Zhu@amd.com>
Date: Wed, 15 Mar 2017 16:36:05 +0800
Subject: [PATCH 0885/2094] drm/amd/powerplay: fix issue dpm blocked by no led
 config.

some cards don't support led config.
so not return error in this case.

Signed-off-by: Rex Zhu <Rex.Zhu@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
---
 drivers/gpu/drm/amd/powerplay/hwmgr/vega10_hwmgr.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/powerplay/hwmgr/vega10_hwmgr.c b/drivers/gpu/drm/amd/powerplay/hwmgr/vega10_hwmgr.c
index 4becd57..43fdbb1 100644
--- a/drivers/gpu/drm/amd/powerplay/hwmgr/vega10_hwmgr.c
+++ b/drivers/gpu/drm/amd/powerplay/hwmgr/vega10_hwmgr.c
@@ -873,7 +873,7 @@ static int vega10_setup_dpm_led_config(struct pp_hwmgr *hwmgr)
 	pp_table->LedPin0 = (uint8_t)(mask & 0xff);
 	pp_table->LedPin1 = (uint8_t)((mask >> 8) & 0xff);
 	pp_table->LedPin2 = (uint8_t)((mask >> 16) & 0xff);
-	return ret;
+	return 0;
 }
 
 static int vega10_setup_asic_task(struct pp_hwmgr *hwmgr)
-- 
2.7.4

